test/src/main/java/org/axonframework/test/fixture/AxonTestPhase.java

```diff
@@ -345,6 +352,19 @@ interface Event {
             Then.Event then();
         }
 
+        /**
+         * When-phase after no action in the When-phase.
+         */
+        interface Nothing {
```

Comment:
Not sure I like plain "nothing", but I don't know anything else. `noActions` suggests commands, `nothingHappens` is to long, `void` is weird, `absence()` isn't correct either... Guess it should be `nothing()`, but wanted to point out it feels slight off.